pkg/feature/templates/serverless/serving-net-istio-secret-filtering.patch.tmpl

```diff
@@ -0,0 +1,21 @@
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: {{ .Serving.Name }}
+  namespace: knative-serving
+spec:
+  deployments:
+    - name: activator
+      annotations:
+        sidecar.istio.io/inject: "true"
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+    - name: autoscaler
+      annotations:
+        sidecar.istio.io/inject: "true"
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+    - name: net-istio-controller
+      env:
+        - container: controller
+          envVars:
+            - name: ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID
+              value: 'true'
```

Comment:
@RetoCode @skonto Tagging you to confirm that this is what makes sense for the long-term.

Reply:
Yes it makes sense. We are working on making this the default anyway. Downstream it will be the default in the next release (1.32).

Please use `workloads` instead of `deployments`. `deployments` is deprecated and replaced with `workloads`.

Reply:
So, I changed, but there is a catch:

Being this a patch, I wonder what would happen for existing installs that need to be upgraded?
My guess is that since this is a patch, we would end up with a resource that would contain both `deployments` and `workloads` (assuming it would be accepted). So, not sure if it would be a good state....

Reply:
We do have that problem either way until we finally drop `deployments`. But `workloads` is applied later than `deployments`, so if we have conflicting values the `workloads` wins.  

Anything I forgot @skonto?

Reply:
Deployments first and then workloads are merged in a slice and then they are applied in that order.